### PR TITLE
🐛 AmpAd: Add clientIdScope to myfinance ad network

### PR DESCRIPTION
**Pull request by pkolenic**
- Configures the clientIdScope for the myfinance Ad Network, which was inadvertently left off initially  
- Fixes attribute typo in the myfinance.md

### CODE REVIEW

File: ads/vendors/myfinance.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {validateData, writeScript} from '../../3p/3p';
+import { validateData, writeScript } from '../../3p/3p';
```
Comment (calebcordry) [inline ads/vendors/myfinance.js:17]: nit: prettier is going to complain about this, you can try `amp lint --local_changes --fix` or follow or docs to setup the formatting extensions in your editor.

**Review by calebcordry — APPROVED** [review]
Thanks for maintaining this extension! Feel free to ping when you fix lint errors and tests go green.

File: ads/vendors/myfinance.js (hunk 1 above)
Comment (pkolenic) [reply ads/vendors/myfinance.js]: Fixed.

**Comment by pkolenic** [comment]
@calebcordry Anything else before it gets merged?

**Pull request merged**